src/components/Kanban/Board.tsx

```diff
@@ -30,6 +32,8 @@ interface KanbanBoardProps<T extends { id: string }> {
   itemRender: (item: T) => ReactNode;
 }
 
+export type KanbanBoardType<T extends { id: string }> = KanbanBoardProps<T>;
+
```

Comment:
if it's going to be exactly same as `KanbanBoardProps`, why not just re-use it?

```suggestion
```

Reply:
> if it's going to be exactly same as `KanbanBoardProps`, why not just re-use it?

Thank you for the suggestion. You're absolutely right that KanbanBoardType<T> is redundant since it's exactly the same as KanbanBoardProps<T>. I'll remove KanbanBoardType<T> and directly reuse KanbanBoardProps<T> to keep the code clean and consistent.